code/game/objects/items/weapons/grenades/flashbang.dm

```diff
@@ -34,4 +34,4 @@
 		M.soundbang_act(1, 200, 10, 15)
 
 	else
-		M.soundbang_act(1, max(20/max(1,distance), 3), rand(0, 5))
+		M.soundbang_act(1, max(200/max(1,distance), 60), rand(0, 5))

```

Comment:
Now he probably means the 3 becoming a 60.
M.soundbang_act(1, max(20/max(1,distance), **3**), rand(0, 5))
M.soundbang_act(1, max(200/max(1,distance), **60**), rand(0, 5))